ci/django.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.11, 3.12, 3.13]
```

Comment:
🤔 I'd recommend the following:

```suggestion
        python-version: ["3.11", "3.12", "3.13"]
```

> [!CAUTION]
> Also please note that `python-versions` after `3.10` and non-`cpython` need quoting to properly function in some GitHub Action Runner Environments (in contrast with `3.7`/`3.8`/`3.9` which do not need quotes); compare with the format used in `ci/pylint.yml` where quotes are used.

### Benefits of these changes:

* **Syntacticly Correct**: Uses quotes around `python-version` correctly.

---

> [!TIP]
> Please see [setup-python action releases](https://github.com/actions/setup-python/releases) Note the changes for version `v5` includes migrating the underlying node version (which fixes a deprecated version just like the 3.7-3.9 to 3.11-3.13 does).

Reply:
Fixed